.github/workflows/black.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint
```

Comment:
Can you rename this file to `format.yml` and rename the action and the job to "Format" accordingly?